src/client/search.js

```diff
@@ -8,7 +8,7 @@ const resultsContainer = document.querySelector("#observablehq-search-results");
 const activeClass = "observablehq-link-active";
 let currentValue;
 
-const index = await fetch(import.meta.resolve("./minisearch.json"))
+const index = await fetch(import.meta.resolve(global.__minisearch))
```

Comment:
We could use the `importMetaResolve` Rollup plugin to say `import.meta.resolve("observablehq:minisearch.json")` here, but that’s just polish.

Reply:
Unlike `/_observablehq/client.js ` which changes only when we update framework, the contents of `/_observablehq/search.js` change about each time we build the site.

Reply:
The latest commit implements `import.meta.resolve` resolution so you can now say `fetch(import.meta.resolve("observablehq:minisearch.json"))`. 🥳